scripts/elemental-airgap.sh

```diff
@@ -384,28 +392,54 @@ build_os_channel() {
     fi
 
     get_chart_val channel_tag "channel.tag"
+    channel_list+="${channel_img}:${channel_tag} "
+
+    # we can have OS channels added in template (mainly for unstable images), so we have to sync them if needed
+    if [[ "$UNSTABLE_CHANNEL" == "true" ]]; then
```

Comment:
With this change only multiple channels are read if `UNSTABLE_CHANNEL` is set to `true`? The expected behavior in the future is that across upgrades we might keep adding channels. Imagine that we have 5.5 channel, Micro 6.0 and Micro 6.1. All those would be managed by Helm and be listed here.

My point is that probably rather than having a flag named `UNSTABLE_CHANNEL` probably it could be named `ALL_CHANNELS` and to my understanding the code would still make sense. I think this might be useful no only for the unstable use case, but also for cases in which the chart already includes other channels.

Consider the current PR under review https://github.com/rancher/elemental-operator/pull/781, with this change it won't be rare that there are more than one channel in addition to the one defined by "channel.repository" as it will attempt to keep old ones too.